backend/cn/pp.mli

```diff
@@ -0,0 +1,177 @@
+(* Module Pp -- Pretty printing *)
+
+(* TODO: BCP: This file could still use some thinning and reorganization! *)
```

Comment:
I suspect one way to proceed might be to have `pp.ml` be just `include PPrint` (or delete it and get users to rename it wherever it is used). Everything else that's currently in `pp.ml` could be turned into `pp_utils.ml`

Reply:
I'll just add your comment to the file, since I dojn't think we're going to do it now.